openedx/core/djangoapps/user_authn/views/registration_form.py

```diff
@@ -431,7 +431,7 @@ def get_registration_form(self, request):
                         required=self._is_field_required(field_name)
                     )
         # remove confirm_email form v1 registration form
-        if is_api_v1(request):
+        if is_api_v1(request) or is_sso_request(request):
```

Comment:
I think we should be consistent in our naming logic. This has been used as is_registration_api_v1 in file openedx/core/djangoapps/user_authn/views/register.py at L174. So, it should be same here.